docs/index.rst

```diff
@@ -198,27 +198,9 @@ The Ocean SDK includes the :ref:`dwave_cli` and the following packages:
 
          :bdg-link-primary:`code <https://github.com/recruit-communications/pyqubo>`
 
-.. packages-end-marker
-
-Index and D-Wave Links
-======================
-
-* :ref:`genindex`: Index for this site.
-
-* `Leap <https://cloud.dwavesys.com/leap>`_: Sign up for Leap quantum cloud service,
-  which gives you immediate, secure access to D-Wave quantum and hybrid solvers, as
-  well as a wealth of information to help you get started creating quantum applications.
-
-* `System Documentation <https://docs.dwavesys.com/docs/latest/index.html>`_: Here you will
-  find information such as
-
-  - `Getting Started  with the System <https://docs.dwavesys.com/docs/latest/doc_getting_started.html>`_---An
-    introduction to D-Wave's quantum computers, their hardware and how they work.
+      .. packages-end-marker
 
-  - `Solver Properties and Parameters <https://docs.dwavesys.com/docs/latest/doc_solver_ref.html>`_---Description
-    of properties and parameters for of D-Wave's quantum computers and
-    Leap's quantum-classical hybrid solvers.
+Index
```

Comment:
might as well get rid of this section, as long it's available in the sidebar

Reply:
It's available in the sidebar as long as the section (its header) exists.